hydrolib/core/io/net/models.py

```diff
@@ -467,22 +525,30 @@ def _link_from_1d_to_2d(
 
         # Note that the function "contacts_compute_multiple" also computes the connections, but does not take into account
         # a bounding polygon or the end points of the 1d mesh.
-        # self._mk.contacts_compute_multiple(self, node_mask)
 
-    def _link_from_2d_to_1d_intersecting(self):
-        raise NotImplementedError()
-
-    def _link_from_2d_to_1d_lateral(self):
-        raise NotImplementedError()
+    def _link_from_2d_to_1d_embedded(
```

Comment:
Is this intended as a public function or private? (given the initial underscore)
If public, consider renaming and adding a detailed docstring.